boot/boot_test.go

```diff
@@ -389,6 +396,25 @@ func (s *bootenv20Suite) TestCurrentBoot20NameAndRevision(c *C) {
 	c.Check(err, Equals, boot.ErrBootNameAndRevisionNotReady)
 }
 
+// only difference between this test and TestCurrentBoot20NameAndRevision is the
+// base bootloader which doesn't support ExtractedRunKernelImageBootloader.
```

Comment:
Thanks for this comment! Maybe reference bootenv20Suite.TestCurrentBoot20NameAndRevision instead of just the short name TestCurrentBoot20NameAndRevision but I guess it's fine from the context.

Reply:
sure I can take this on in a followup